### PR TITLE
Add missing end points to path

### PR DESCRIPTION
**Pull request by mikeybyker**
...to the Collapsing and Commitment scenarios

### CODE REVIEW

File: js/scenarios.js [hunk 1]
```diff
@@ -129,7 +129,8 @@ var Scenarios = [
       {x: 1, y: 2},
       {x: 7, y: 0},
       {x: 7, y: 2},
-      {x: 7, y: 1}
+      {x: 7, y: 1},
+      {x:8, y:1}
```
Comment (epatrasc) [inline js/scenarios.js]: can you correct this line?
{x: 8, y: 1}
😄 (space)

Comment (mikeybyker) [reply js/scenarios.js]: OK, added! But note that the spacing is inconsistent throughout the file...

Comment (epatrasc) [reply js/scenarios.js]: yeah, you are right, now I'm noticing it. I will clean-up  and push the changes.
Thanks

**Comment by epatrasc** [comment]
Can we merge this? I will add the spacing refactoring on my branch...

**Comment by mikeybyker** [comment]
You can as far as I am concerned (...I used the same changes in my own branch)
I left the spacing, formatting etc. alone in all of the original files. I just use Prettier on all my own, takes away all this formatting pain :-)

**Pull request merged**